house/models.py

```diff
@@ -48,5 +58,19 @@ class House(models.Model):
     income = models.IntegerField(null=True, blank=True)
     children = models.IntegerField(null=True, blank=True)
 
+    @staticmethod
+    def create_house(house_id, name, public, country, city, parent_profession_1, parent_profession_2, income, children):
```

Comment:
Do we need this method? Isn't it built-in with Django models?

Reply:
the Django model create function I.E {model}.object.create is not suitable for test creation, but will still be used in migrations and app flow creation. but for local creation and testing this will be a good helper function